.pre-commit-config.yaml

```diff
@@ -12,4 +12,4 @@ repos:
   rev: v1.12.0
   hooks:
   - id: blacken-docs
-    additional_dependencies: [black==21.11b1]
+    additional_dependencies: [black==22.3.0]
```

Comment:
Need https://github.com/4Catalyzer/fourmat/pull/39 for above fourmat call also I think